.NET/Microsoft.Recognizers.Text.Number/Models/AbstractNumberModel.cs

```diff
@@ -9,26 +9,35 @@ namespace Microsoft.Recognizers.Text.Number
     public abstract class AbstractNumberModel : IModel
     {
         // Languages supporting subtypes in the resolution to be added here
-        private static readonly List<string> ExtractorsSupportingSubtype = new List<string> { Constants.ENGLISH, Constants.SWEDISH, Constants.ARABIC };
+        private static readonly List<string> ExtractorsSupportingSubtype = new List<string> { Constants.ENGLISH, Constants.SWEDISH, Constants.ARABIC, Constants.JAPANESE_SUBS };
 
         protected AbstractNumberModel(IParser parser, IExtractor extractor)
         {
             this.Parser = parser;
             this.Extractor = extractor;
         }
 
+        protected AbstractNumberModel(IParser parser, IExtractor extractor, bool recode)
+        {
+            this.Parser = parser;
+            this.Extractor = extractor;
+            this.Recode = recode;

```

Comment:
What's the purpose of recode?
Japanese was following the Chinese framework and both didn't require this previously.